mmv1/templates/terraform/examples/tpu_v2_queued_resource_network_configs.tf.tmpl

```diff
@@ -0,0 +1,67 @@
+resource "google_tpu_v2_queued_resource" "{{$.PrimaryResourceId}}" {
+  provider = google-beta
+
+  name    = "{{index $.Vars "qr_name"}}"
+  zone    = "us-central1-c"
+  project = "{{index $.TestEnvVars "project"}}"
+
+  tpu {
+    node_spec {
+      parent  = "projects/{{index $.TestEnvVars "project"}}/locations/us-central1-c"
+      node_id = "{{index $.Vars "tpu_name"}}"
+      node {
+        runtime_version  = "tpu-vm-tf-2.13.0"
+        accelerator_type = "v2-8"
+
+        network_configs = [
+	  {
+            can_ip_forward      = true
+            enable_external_ips = true
+            network             = google_compute_network.network_0.id
+            subnetwork          = google_compute_subnetwork.subnet_0.id
+	    queue_count         = 32
+          },
+	  {
+            can_ip_forward      = true
+            enable_external_ips = true
+            network             = google_compute_network.network_1.id
+            subnetwork          = google_compute_subnetwork.subnet_1.id
+	    queue_count         = 32
+          }
+	]
```

Comment:
Missing test detection relies on lists using the repeated notation (instead of the array notation)

```suggestion
        network_configs {
            can_ip_forward      = true
            enable_external_ips = true
            network             = google_compute_network.network_0.id
            subnetwork          = google_compute_subnetwork.subnet_0.id
	    queue_count         = 32
	}
        network_configs {
            can_ip_forward      = true
            enable_external_ips = true
            network             = google_compute_network.network_1.id
            subnetwork          = google_compute_subnetwork.subnet_1.id
	    queue_count         = 32
	}
```